dev_requirements.txt

```diff
@@ -1,9 +1,6 @@
-dbt-core==0.19.0
-pyodbc>=4.0.27
-azure-identity>=1.4.0
 black~=20.8b1
 pytest-dbt-adapter~=0.4.0
+pytest~=6.2.2
 tox==3.2.0
 flake8>=3.5.0
 certifi==2020.6.20
```

Comment:
is this used for development or for the package itself?

Reply:
not as far as I can tell. might as well try removing it!
```suggestion
```